native/cocos/renderer/GFXDeviceManager.h

```diff
@@ -150,7 +150,7 @@ class CC_DLL DeviceManager final {
         }
 
 #if !defined(CC_SERVER_MODE)
-        if (CC_DEBUG > 0 && !FORCE_DISABLE_VALIDATION || FORCE_ENABLE_VALIDATION) {
+        if (CC_DEBUG > 0 && (!FORCE_DISABLE_VALIDATION || FORCE_ENABLE_VALIDATION)) {
```

Comment:
I think it should be `CC_DEBUG > 0 && !FORCE_DISABLE_VALIDATION || FORCE_ENABLE_VALIDATION`

Reply:
Fixed